xbmc/addons/Addon.cpp

```diff
@@ -622,6 +621,52 @@ AddonVersion CAddon::GetDependencyVersion(const std::string &dependencyID) const
   return AddonVersion("0.0.0");
 }
 
+
+void OnPreInstall(const AddonPtr& metaAddon)
```

Comment:
Can´t follow. I just don´t like `metaAddon` as name for the addon pointer that is passed. Why can´t it just be `addon`?